### PR TITLE
Add group write perm to django log files

### PR DESCRIPTION
**Pull request by a-milogradov**
in context of https://github.com/yola/production/issues/6195

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: yodeploy/hooks/django.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def prepare_logfiles(self):
                 continue
             with ignoring(errno.EEXIST):
                 os.mkdir(os.path.dirname(logfile))
-            touch(logfile, self.log_user, self.log_group, 0o640)
+            touch(logfile, self.log_user, self.log_group, 0o660)
```
Comment (Toshakins) [inline yodeploy/hooks/django.py:46]: What's the reason behind it?

Comment (a-milogradov) [reply yodeploy/hooks/django.py:46]: please, see attached ticket

Comment (deonbredenhann) [reply yodeploy/hooks/django.py:46]: I don't have permission to see the ticket. :disappointed:

Comment (Toshakins) [reply yodeploy/hooks/django.py:46]: @deonbredenhann We want to enable people to run `./manage.py shell` without `sudo` :)

Right now mere developers don't have access to write to the log file.

Comment (deonbredenhann) [reply yodeploy/hooks/django.py:46]: 👍 😄

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Pull request merged**